### PR TITLE
Updates to import script for moving content from CB to LB

### PR DESCRIPTION
**Pull request by dmcavoy**
While was importing AI ML into CSD (and when I did the test PL import) I noticed some things about the import script which have not been updated as we have made changes based on feedback. This tries to add some of the easier wins of updating the import script. There are still other things we will need to update that I did by hand here.

Updated:
- We couldn't use fix_script_level_positions because of the new error when you try to run it on non-migrated script. I added a parameter to the method so it will know when you are importing and allow it for that case.
- Keys for resources and vocab need to be downcase and use underscores because of the new validation on those keys
- We need to save the lesson overviews in the i18n information after they are imported
- Progression names need to be stored in the new progression name fields instead of the activity section name

## Testing story

I made these updates and then used them to import the AI ML unit. 

## Follow-up work

Still needs to be updated:
- Activity section for the last progression still has progression name as the activity section title
- We don't pull over unit overview or unit titles. Do we want to?
- Code blocks don't get updated to the new syntax during the import
-  You have to save the script for resources and vocab roll-up pages to be added by the add rollup pages button. Probably need to save the script somewhere in the import?
- Need to update resource and vocab inline markdown syntax through import to the new format with course version information
Tracked in: https://codedotorg.atlassian.net/browse/PLAT-1026

### CODE REVIEW

File: bin/curriculum/import_unit_details.rb [hunk 1]
```diff
@@ -127,7 +127,7 @@ def main(options)
       lesson_group.update_from_curriculum_builder(cb_chapter) if options.models.include?('LessonGroup')
     end
 
-    script.fix_script_level_positions
+    script.fix_script_level_positions(true)
     script.update!(show_calendar: !!cb_unit['show_calendar'], is_migrated: true)
```
Comment (bethanyaconnor) [inline bin/curriculum/import_unit_details.rb:130]: Could you just flip these two lines instead of adding a parameter to `fix_script_level_positions`?

Comment (davidsbailey) [reply bin/curriculum/import_unit_details.rb:130]: +1

**Review by davidsbailey — APPROVED** [review]
Nice fixes Dani! Thank you for capturing the follow-up work. My overall thought there is that we should balance the amount of effort we spend on those with the number of times this script will be run in the future.

File: bin/curriculum/import_unit_details.rb (hunk 1 above)
Comment (dmcavoy) [reply bin/curriculum/import_unit_details.rb:130]: Done

**Review by bethanyaconnor — APPROVED** [review]

**Pull request merged**